app/components/navbar.tsx

```diff
@@ -47,7 +47,7 @@ const Navbar: React.FC = () => {
 			label: "Open Source",
 		},
 		{ label: "Feedback" },
-		{ href: `${homepageUrl}/settings`, label: "Settings" },
+		viewer.role === 'admin' || viewer.role ===  'owner' ? { href: `${homepageUrl}/settings`, label: "Settings" } : {},
```

Comment:
can we use a constant instead ? For example showSettingLink = viewer.role === 'admin' ||  'owner' . Also I think it will be more clear to add a field for the each object = show: true, { href: `${homepageUrl}/settings`, label: "Settings", show: showSettingLink},  { href: `${homepageUrl}/sign_out`, label: "Sign Out" , show: true}
and while rendering render it like link.href && link.show (....

Reply:
yes I will make this change